### DISCUSSION TITLE
Eliminate page break option.

### DISCUSSION THREAD

**Discussion by dougie-wougie** [code4recovery/12-step-meeting-list#1173] [Ideas]
I would like https://pdf.code4recovery.org/ to add a checkbox and code to eliminate the page breaks between the days. This would save us a few sheets of page with each printing.

doug

**Comment by joshreisner** [comment]
thanks -  you should expect a reply soon on your ticket: https://github.com/code4recovery/pdf/issues/47 - you posted that in the right place
